pkg/minishift/registration/utils.go

```diff
@@ -60,6 +61,10 @@ func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
 		if err := registrator.Register(param); err != nil {
 			return err
 		}
+		minishiftConfig.InstanceConfig.Register = true
+		if err := minishiftConfig.InstanceConfig.Write(); err != nil {
+			return err
```

Comment:
Hmm, I am wondering whether one should not just log a warning and continue. Is this really an abort situation? Not saying it is not, but I think we also should start asking ourselves once in a while whether each error leads to failure. It might be enough to log a warning or try to recover. Not sure.